.env

```diff
@@ -1,3 +1,3 @@
-WEB_SITE_URL = "https://tech-stack.io"
```

Comment:
Please, dont use .env as a storage for links, create appsetting.json to store such data, .env is technical tool for extending environment variables, that's not his usage ❌❌❌❌❌❌❌❌❌❌❌❌❌❌❌❌❌❌❌❌